build/media-suite_compile.sh

```diff
@@ -2841,6 +2841,12 @@ if [[ $mpv != n ]] && pc_exists libavcodec libavformat libswscale libavfilter; t
         if ! mpv_disabled manpage-build || mpv_enabled html-build; then
             do_pacman_install python-docutils
         fi
+        if enabled libnpp && [[ -n "$CUDA_PATH" ]]; then
+            mpv_cflags=("-I$(cygpath -sm "$CUDA_PATH")/include")
+            mpv_ldflags=("-L$(cygpath -sm "$CUDA_PATH")/lib/x64")
+            MPV_OPTS["c_args"]="$(IFS=,; echo "${mpv_cflags[*]}")"
```

Comment:
The reason I mentioned these lines was in consideration of someone having c flags already here. This would completely override them.